### ISSUE TITLE
Expose tooltipTitle in all charts

### ISSUE THREAD

**Issue by mcrascal** [evidence-dev/evidence#413]
**Feature Description**
Add the option for tooltipTitle to all charts (e.g. LineChart, BarChart) etc, per: https://github.com/evidence-dev/evidence/pull/399

**Goal of Feature**
User wants to include period over period metrics as additional information on a line chart. So hovering over a specific date would show change from a prior date to that date. Many other similar use cases. 

**Current Solution**
What do you do today to accomplish the goal you described above?
Separate chart, or mixed chart

**Are you interested in contributing this feature?**
Yes

**Comment by archiewood** [comment]
@mcrascal I also feel the need for this kind of additional information

But there is often already a title in a barchart's tooltip. Would the idea be to override this?

<img width="719" alt="image" src="https://user-images.githubusercontent.com/58074498/190524043-449faa04-ab1d-44f9-994d-b4fbc47f5259.png">


How often would this title field be preferable to either:
- Another field in the tooltip, but not as title
- another series (eg ability to add a line for yoy growth)

**Comment by mcrascal** [comment]
> There is often already a title in eg a barchart's tooltip. Would the idea be to override this?

Yes, idea is the functionality would be the exact same as in scatter plot. 

> How often would this title field be preferable to either:
> * Another field in the tooltip, but not as title
> * another series (eg ability to add a line for yoy growth)

I think another field in the tooltip is a good idea, but it'd be a net new feature in addition to this, while this is just bringing the rest of the charts to parity w/ the scatter plot. 

I think another series often would be better, but there are going to be cases where this would be handy.
